cmd/gateway/commands.go

```diff
@@ -198,6 +212,28 @@ func createStaticModeCommand() *cobra.Command {
 		"Update the status of the GatewayClass resource.",
 	)
 
+	cmd.Flags().BoolVar(
+		&disableMetrics,
+		"metrics-disable",
+		false,
+		"Disable exposing metrics in the Prometheus format.",
+	)
+
+	cmd.Flags().IntVar(
```

Comment:
can we also add validation test case here https://github.com/nginxinc/nginx-kubernetes-gateway/blob/bb0157ef210bf72e47197e391456da81b6a988a6/cmd/gateway/commands_test.go#L113 ?

(sorry, missed it during the previous review)